### DISCUSSION TITLE
Note button on ipad

### DISCUSSION THREAD

**Discussion by colindavey** [vincode-io/Zavala#264] [General]
Hi, I’m finding that the note button is putting the note on the wrong item. When the cursor is in an item, and I hit the note button, the note gets created under a different item. As far as I can tell, the note gets added to a different item which was recently edited. 

I’m running on an iPad and I don’t know if it’s specific to that platform.

**Comment by colindavey** [comment]
I made an issue for this issue,

**Comment by vincode-io** [comment]
This was resolved in the 3.0.5 release.
